src/graphql/query/teamsGoals.graphql

```diff
@@ -1,5 +1,5 @@
 query GetGoals($teamId: Int, $limit: Int) {
-	getGoals(limit: $limit, filters: { audienceType: TEAM, isActive: true, teamId: $teamId}) {
+	getGoals(limit: $limit, filters: { audienceType: TEAM, isActive: true, teamId: $teamId, status: [IN_PROGRESS]}) {
```

Comment:
We might want to future proof by adding `NONE`, since that status still might be used.

Reply:
```suggestion
	getGoals(limit: $limit, filters: { audienceType: TEAM, isActive: true, teamId: $teamId, status: [IN_PROGRESS, NONE]}) {
```

Reply:
one thing to note, from what I've seen goals that are intended for the future (this is based on integration test cases) are in the status NONE. I'm wondering if we should/could get rid of the isActive field in the future and just use this status, as they seem a bit redundant and if we fail to update one or the other at some point we may face some data consistency issues.

Reply:
The `NONE` status was created, but we didn't add code to change that to `IN_PROGRESS` in the service. So we started the concept of using `NONE`, but some work remained. One different is `isActive` is that it checks the dates on the goal, so that is still important.